### ISSUE TITLE
Rename artifactId to kcctl

### ISSUE THREAD

**Issue by aalmiray** [kcctl/kcctl#25]

**Comment by gunnarmorling** [comment]
Ah yes, good point. Also, the group id should be "org.kcctl", I just reserved the kcctl.org domain name. I suppose I need to jump through some hoops to get that as a group id on Maven Central, right?

**Comment by aalmiray** [comment]
That is correct. You'll have to file a ticket https://andresalmiray.com/publishing-to-maven-central-using-apache-maven/

**Comment by gunnarmorling** [comment]
@aalmiray, do we need to do any changes to the JReleaser config when changing the artifact id?

**Comment by aalmiray** [comment]
No, there’s no need. That’s the beauty of using the Maven plugin and convention-over-configuration, if the `artifactId` and `groupId` were to be used in any way by JReleaser their values would be taken directly from those defined in the pom at their standard location. Currently those values are not needed, you’re good to go.

**Comment by gunnarmorling** [comment]
Done.
